.gitattributes

```diff
@@ -1,4 +1,4 @@
 *.go -text diff=golang
 
 lib/runtime/test_data/kusama/ filter=lfs diff=lfs merge=lfs -text
-lib/runtime/test_data/polkadot/ filter=lfs diff=lfs merge=lfs -text
+lib/runtime/test_data/polkadot/ filter=lfs diff=lfs merge=lfs -text
```

Comment:
Please configure your IDE to trim whitespaces and add a new line at the end of each file (for consistency).